repositories.yaml

```
@@ -43,6 +46,7 @@ reference-browser:
     - org.mozilla.components:lib-crash
 fenix-nightly:
   app_id: org-mozilla-fenix-nightly
+  description: Firefox for Android
```

For these types of cases, where one repo corresponds to multiple apps, what do we do? We could probably just do something like:

```suggestion
  description: Firefox for Android (nightly version)
```

While we're here it would be nice to sort out which versions of Fenix and other products are still valid. I might want to explicitly tag deprecated stuff in the future, but at least updating the description would be a start.

---

Yeah. On second thought, what do you think about adding an (optional) deprecated field right away? It would be nice to have an authoritative way of marking a repository as deprecated.

---

And actually, looking through it again-- how about an (optional) channel field?

---

+1 on channel. It fits our deploy model, no reason to exclude it because it's specific.

---

Ok, I'll go ahead and add channel and deprecated to the schema and set it where appropriate.